File: KIWI/Model/ConversionLog.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace KIWI {
    public class ConversionLog {
        [Column("[orderNumber]")]
        public int orderNumber { get; set; }
        [Column("[shipTo]")]
        public int shipTo { get; set; }
        [Column("[item]")]
        public int item { get; set; }
        [Column("[material]")]
        public int material { get; set; }
        [Column("[oldQty]")]
        public int oldQty { get; set; }
        [Column("[newQty]")]
        public int newQty { get; set; }
        [Column("[isConverted]")]
        public bool isConverted { get; set; }
        [Column("[startTime]")]
        public DateTime startTime { get; set; }
        [Column("[endTime]")]
        public DateTime endTime { get; set; }
        [Column("[status]")]
        public string status { get; set; }
        [Column("[isSaved]")]
        public bool isSaved { get; set; }
    }
}

File: KIWI/Model/ConversionLog.vb
Imports System.ComponentModel.DataAnnotations.Schema

Public Class ConversionLog

    <Column("[orderNumber]")>
    Public Property orderNumber As Integer
    <Column("[shipTo]")>
    Public Property shipTo As Integer
    <Column("[item]")>
    Public Property item As Integer
    <Column("[material]")>
    Public Property material As Integer
    <Column("[oldQty]")>
    Public Property oldQty As Integer
    <Column("[newQty]")>
    Public Property newQty As Integer
    <Column("[isConverted]")>
    Public Property isConverted As Boolean
    <Column("[startTime]")>
    Public Property startTime As Date
    <Column("[endTime]")>
    Public Property endTime As Date
    <Column("[status]")>
    Public Property status As String
    <Column("[isSaved]")>
    Public Property isSaved As Boolean

End Class
